packages/utils/package.json

```
@@ -2,11 +2,17 @@
   "name": "@veramo/utils",
   "description": "Helper methods for Veramo plugins",
   "version": "5.6.0",
-  "main": "build/index.js",
-  "exports": "./build/index.js",
+  "main": "build/index.cjs.js",  
+  "module": "build/index.esm.js",
   "types": "build/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./build/index.esm.js",
```

I think this should also export types
```suggestion
      "types": "./build/index.d.ts",
      "import": "./build/index.esm.js",
```